app/Http/Controllers/LocationsController.php

```diff
@@ -264,5 +264,27 @@ public function show($locationId = null)
 
         return redirect()->route('locations.index')->with('error', trans('admin/locations/message.does_not_exist', compact('id')));
     }
+    
+    public function printOnlyAssignedToLocation($id)
+    {
+
+        $show_location = Location::where('id',$id)->withTrashed()->first();
+	$manager = User::where('id',$show_location->manager_id)->withTrashed()->first();
+	$users = User::where('location_id', $id)->with('company', 'department', 'location')->get();
+        $assets = Asset::where('assigned_to', $id)->where('assigned_type', Location::class)->with('model', 'model.category')->get();
+        return view('locations/print')->with('assets', $assets)->with('users',$users)->with('show_location', $show_location)->with('manager', $manager);
+
+    }
+    
```

Comment:
Docblocks here too please :) I know you know what you're trying to do here, but in a year, no one else will :)